CalibTracker/SiStripChannelGain/python/ntuple_cff.py

```diff
@@ -12,14 +23,53 @@
                                      'keep *_shallowTracks_tracketa_*',
                                      'keep *_shallowTracks_trackphi_*',
                                      'keep *_shallowTracks_trackhitsvalid_*',
+                                     'keep *_shallowTracks_trackalgo_*',
                                      'keep *_shallowGainCalibration_*_*'
```

Comment:
the tracking iteration is now used to filter on good tracks for the gain calibration. Hence trackalgo has to be delivered in the calibration ntuple.